examples/convert.rs

```diff
@@ -157,6 +157,19 @@ fn main() {
 
             fs::write(&args[2], bytes.as_slice()).unwrap();
         }
+        #[cfg(feature = "glsl450-backend")]
+        "vert" | "frag" => {
+            use naga::back::glsl450;
```

Comment:
let's not call the module "glsl450", since, as we discussed, we need a more generic glsl backend than just 450

Reply:
I named it glsl450 because the code that it has can only generate glsl 450 and 460 and since most of the code is already in glsl-common we can add other versions and disable them trough features to reduce code size and compile time

Reply:
We need features to turn on/off big chunks of code, which is what mostly cared by browsers and WASM targets. Do you think it makes sense to turn GLSL450 output separately ON from GLSLES300 ? My understanding is that the latter is much more involved, but also is what browsers would need, so at least the users I mentioned don't need GLSL450 feature to be separate. Would your app need it?

Reply:
No i dont need it for any particular reason if you think it is okay to bundle them all together than for me it's okay

Reply:
yeah, I think it's fine to have both behind `glsl-out` feature